docs/user-guide/rules/http-cache.md

```diff
@@ -279,6 +279,52 @@ Also note that:
 
 </details>
 
+<details>
+<summary>How to configure IIS</summary>
+
+You can enable the `Cache-Control` and/or `Expire` headers on IIS using
+the [`<clientCache> element under <staticContent>`][clientcache iis(https://docs.microsoft.com/en-us/iis/configuration/system.webserver/staticcontent/clientcache)].
+
+`<clientCache>` will set the cache for all the configured static content
+so you might want to use it in combination with the `<location>` element
+and set different values depending on where the resources are in the file
+system.
+
+The following is an example that sets `cache-control: no-cache` for all
+static resources and then overrides it for the files under the `static` folder
+with `cache-control: max-age=31536000, immutable`:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <system.webServer>
+        <staticContent>
+            <clientCache cacheControlMode="DisableCache" />
+        </staticContent>
+    </system.webServer>
+    <location path="static">
+        <system.webServer>
+            <staticContent>
+                <clientCache cacheControlMode="UseMaxAge" cacheControlMaxAge="365.00:00:00" cacheControlCustom="immutable" />
+            </staticContent>
+        </system.webServer>
+    </location>
+</configuration>
+```
+
+In this case, you would want to have your JavaScript, CSS, images, etc.
+under `/static`, and your HTML elesewhere.
```

Comment:
Maybe say something in the lines of: If your content is not under `static/`, you will need to change the `<location path="static">` part and point it to the right directory?